ORM/File.cs

```
@@ -12,21 +13,98 @@ class File
 
         private string path;
         private List<Person> people = new List<Person>();
-        private Person person;
-
+  
         public string Path => path;
-        public Person Person => person;
+
+        public int Count => people.Count;
 
         public List<Person> People => people;
 
         public File(string path, Person person)
         {
             this.path = path;
-            this.person = person;
+            people.Add(person);
+  
+        }
 
+        /// <summary>
+        /// add new person for listpeople
+        /// </summary>
+        /// <param name="person"></param>
+        public void Create(Person person)
+        {
+            int index = people.FindIndex(p => p.Id == person.Id);
+            if(index == -1)
+                people.Add(person);
+            else
+                Console.WriteLine("user with this id already registered");
```

Lets throw some kind of exception instead of printing in console